istio/istio.spec

```diff
@@ -521,6 +521,9 @@ sort -u -o devel.file-list devel.file-list
 %endif
 
 %changelog
+* Mon Jun 17 2019 Jonh Wendell <bavery@redhat.com> - 0.12.0-3
```

Comment:
You may want to update the email address :)

Reply:
lol

Reply:
I just wanted to be Brian for 1 day, you ruined my plans